skimage/transform/tests/test_thin_plate_splines.py

```diff
@@ -10,58 +10,61 @@
 
 
 class TestThinPlateSplineTransform:
+    tform_class = ThinPlateSplineTransform
```

Comment:
Curious about the reason for the class attribute here? Did you plan to inherit from this class? Otherwise I'm not really seeing a reason for the indirection. 

But fine with me and not a blocker.